src/styles/default-theme.js

```diff
@@ -8,11 +8,143 @@ import { css } from 'lit-element';
+    /*endregion*/
+
+    /*region Color Decisions (background)*/
+    /* Usage: Usually interactive, used for sensitive actions.
+    -- Use this with --color-text-inverted --
+    For instance: delete / remove buttons. */
+    --color-bg-danger: var(--color-red);
+
+    /* Usage: default background color
+    -- Not used at the moment - will be when we allow theme override. -- */
+    --color-bg-default: var(--color-white);
+
+    /* Usage: neutral background that stands out a little bit from the default one.
+    For instance: blocks, cards, table rows. */
+    --color-bg-neutral: var(--color-grey-10);
+
+    /* Usage: another shade neutral background that stands out a little bit from the default background and neutral one.
+    For instance: blocks, cards, table rows. */
+    --color-bg-neutral-alt: var(--color-grey-15);
+    
+    /* Usage: element with a neutral background with active status.
+    For instance: When being clicked, + / - button backgrounds inside cc-input-text */
+    --color-bg-neutral-active: var(--color-grey-20);
+
+    /* Usage: element with neutral background is disabled.
+    For instance: disabled text / number input */
+    --color-bg-neutral-disabled: var(--color-grey-15);
+    
+    /* Usage: element with a neutral background is hovered.
+    For instance: Card or table row being hovered. */
+    --color-bg-neutral-hovered: var(--color-grey-15);
+
+    /* Usage: element with a neutral background is hovered.
+    For instance: Card or table row being hovered. */
+    --color-bg-neutral-readonly: var(--color-grey-10);
+
+    /* Usage: Usually interactive, used to trigger basic actions that are not submit, delete, reset, etc.
+    -- Use this with --color-text-inverted --
+    For instance: Modal opening button, add an element without refreshing the page. */
+    --color-bg-primary: var(--color-blue);
+
+    /* Usage: notice ? */
+    --color-bg-primary-light: var(--color-blue-light);
+
+    /* Usage: content that needs to stand out a little more than primary content.
+    For instance: some icons. */
+    --color-bg-primary-highlight: var(--color-blue-highlight);
+    
+    /* Usage: content that needs a plain but very soft background.
+    For instance: tags, pieces of text that need to stand out but not too much */
+    --color-bg-soft: var(--color-purple-light);
+    
+    /* Usage: header background when something needs to stand out
+    -- Use this with --color-text-inverted -- 
+    For instance: table header, ribbon. */
+    --color-bg-strong: var(--color-blue-dark);
+
+    /* Usage: Usually interactive, used to trigger submit forms / data, validate something.
+    -- Use this with --color-text-inverted --
+    For instance: submit buttons */
+    --color-bg-success: var(--color-green);
+    
+    /* Usage: elements containing short texts with a "caution" / "warning" meaning and usually interactive.
+    For instance: ? */
+    --color-bg-warning: var(--color-orange);
+    /*endregion*/
   }
 `;
```

Comment:
The whole file is nice:

* foldable sections :+1: 
* usage docs :+1: 
* naming of variables :+1: